tests/test_collection.py

```diff
@@ -1201,3 +1191,28 @@ def test_driver_detection(tmpdir, extension, driver):
         crs="EPSG:4326",
     ) as output:
         assert output.driver == driver
+
+
+@pytest.mark.skipif(
+    sys.platform == "win32",
+    reason="Windows test runners don't have full unicode support",
```

Comment:
We'll figure this out later. Some lack of libiconv or configuration thereof, I bet.